include/alpaka/acc/AccOacc.hpp

```diff
@@ -0,0 +1,511 @@
+/* Copyright 2020 Jeffrey Kelling
+ *
+ * This file is part of Alpaka.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#pragma once
+
+#ifdef ALPAKA_ACC_ANY_BT_OACC_ENABLED
+
+#if _OPENACC < 201306
+    #error If ALPAKA_ACC_ANY_BT_OACC_ENABLED is set, the compiler has to support OpenACC xx or higher!
+#endif
+
+// Base classes.
```

Comment:
Comments like these might be scrambled by clang-format's header sorting. I would remove them.

Reply:
This is also a copy from OpenMP. Could be addressed when clang format is there and a decision has been made on wether they should be there or not also for other backends.